frontend/packages/component-library/src/cms/heroBanner/heroBanner.module.scss

```
@@ -0,0 +1,93 @@
+@use '@code-dot-org/component-library-styles/colors.scss';
+@use '@code-dot-org/component-library-styles/variables' as variables;
+@use '@code-dot-org/component-library-styles/typography.module' as typography;
+
+$mobileBreakpoint: 1024px;
```

I use this on the existing hero banners, we want the two column design to stay on small desktops.
```suggestion
$mobileBreakpoint: 970px;
```

---

cc @moshebaricdo what do you think?

---

Makes sense to me so we can keep 2-col for small desktop, do we use the 970 breakpoint elsewhere?

---

I see this is ussed in 55 files 77 times across code.org codebase